plan/plans/show.go

```diff
@@ -111,6 +120,11 @@ func (s *ShowPlan) GetFields() []*field.ResultField {
 		names = []string{"Table", "Create Table"}
 	case stmt.ShowGrants:
 		names = []string{fmt.Sprintf("Grants for %s", s.User)}
+	case stmt.ShowTriggers:
+		names = []string{"Trigger", "Event", "Table", "Statement", "Timing", "Created",
+			"sql_mode", "Definer", "character_set_client", "collation_connection", "Database Collation"}
```

Comment:
https://dev.mysql.com/doc/refman/5.7/en/show-triggers.html
I copy the column names from here. It is "sql_mode".